eos/saveddata/fighter.py

```diff
@@ -53,6 +53,20 @@ def __init__(self, item):
 
         self.build()
 
+        standardAttackActive = False
+        for ability in self.abilities:
+            if ability.effect.isImplemented and ability.effect.handlerName == 'fighterabilityattackm':
+                # Activate "standard attack" if available
+                ability.active = True
+                standardAttackActive = True
+            else:
+                # Activate all other abilities (Neut, Web, etc) except propmods if no standard attack is active
+                if ability.effect.isImplemented and \
+                                standardAttackActive is False and \
+                                ability.effect.handlerName != 'fighterabilitymicrowarpdrive' and \
+                                ability.effect.handlerName != 'fighterabilityevasivemaneuvers':
+                    ability.active = True
+
```

Comment:
Moved this bit from the add fighter function to the actual fighter initialization